pkg/uroot/util/uname.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2015-2017 the u-root Authors. All rights reserved
```

Comment:
I thought all the stuff in x/unix replaced this? I have not looked much however.

Reply:
x/unix doesn't have a Uname for darwin, so we still need the darwin/linux split.

Reply:
FWIW I just sent https://golang.org/cl/77270 which adds Uname for darwin. Once the CL is approved, x/sys/unix could be re-vendored and would make this wrapper unnecessary.

Reply:
Nice @tklauser